docs/content/tutorial/step_09.ngdoc

```diff
@@ -38,16 +38,16 @@ Since we are using [Bower][bower] to install client-side dependencies, this step
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "angular": "1.5.x",
-    "angular-mocks": "1.5.x",
-    "angular-route": "1.5.x",
+    "angular": "1.6.x",
+    "angular-mocks": "1.6.x",
+    "angular-route": "1.6.x",
```

Comment:
In the tutorial, it was `2.2.x` --> `3.2.x`. Is 1.5.x compatible with jQuery 3.x?